Lib/test/test_functools.py

```diff
@@ -3128,7 +3134,10 @@ def test_access_from_class(self):
         self.assertIsInstance(CachedCostItem.cost, py_functools.cached_property)
 
     def test_doc(self):
-        self.assertEqual(CachedCostItem.cost.__doc__, "The cost of the item.")
+        self.assertEqual(CachedCostItem.cost.__doc__,
+                         ("The cost of the item."
+                          if support.HAVE_DOCSTRINGS
+                          else None))
```

Comment:
Or should this whole test just be skipped if we're in `-00` mode, maybe?

Reply:
I think that checking that docstring is stripped in `-OO` is also helpful. So, I prefer to check it as well.

Reply:
Sure. I don't have a strong opinion either way -- feel free to merge :)